client/mainmenu/CPrologEpilogVideo.cpp

```diff
@@ -29,8 +29,7 @@ CPrologEpilogVideo::CPrologEpilogVideo(CampaignScenarioPrologEpilog _spe, std::f
 
 	CCS->videoh->open(spe.prologVideo);
 	CCS->musich->playMusic(spe.prologMusic, true, true);
-	// MPTODO: Custom campaign crashing on this?
-//	voiceSoundHandle = CCS->soundh->playSound(CCampaignHandler::prologVoiceName(spe.prologVideo));
+	voiceSoundHandle = CCS->soundh->playSound(spe.prologVoice);
```

Comment:
Will this work with custom campaigns? No "failed to play sound" errors in console or something like that?